core/src/test/java/ai/timefold/solver/core/impl/solver/SolverSolutionInheritanceTest.java

```diff
@@ -38,11 +38,11 @@ void testBothClassesAnnotated() {
                 .withSolutionClass(TestdataBothAnnotatedExtendedSolution.class)
                 .withEntityClasses(TestdataBothAnnotatedChildEntity.class)
                 .withConstraintProviderClass(TestdataBothAnnotatedExtendedConstraintProvider.class)
-                .withTerminationConfig(new TerminationConfig().withMoveCountLimit(14L));
+                .withTerminationConfig(new TerminationConfig().withMoveCountLimit(100L));
 
         var problem = TestdataBothAnnotatedExtendedSolution.generateSolution(3, 2);
         var solution = PlannerTestUtils.solve(solverConfig, problem);
-        assertThat(solution.getScore()).isEqualTo(SimpleScore.of(8));
+        assertThat(solution.getScore()).isEqualTo(SimpleScore.of(16));
```

Comment:
Another change in behavior. Although if this is just due to a change in the score calculator, I'm OK with it.